### ISSUE TITLE
No urequests included?

### ISSUE THREAD

**Issue by chriskalv** [sbcshop/Pico-WiFi-HAT#3]
The main reason for a WiFi hat including a display is probably pulling data from the internet and displaying it in some way. 

Hence, I have no idea, why the urequests library is not included here.

If you can, please fix this.

**Comment by midnightsurfer** [comment]
Indeed, this seems odd. If there is an alternative urequests library then I would love to hear about it as having trouble locating one that works on the Raspberry Pico with this WiFi hat for some reason but then I am new to this.
